packages/web-pkg/tests/unit/components/FilesList/ResourceTable.spec.ts

```diff
@@ -351,7 +351,9 @@ describe('ResourceTable', () => {
       const tr = await wrapper.find('.oc-tbody-tr-forest .oc-resource-name')
       await tr.trigger('click')
 
-      expect(wrapper.emitted().fileClick[0][0].resources[0].name).toMatch('forest.jpg')
+      expect(
+        (wrapper.emitted('fileClick')[0][0] as { resources: Resource[] }).resources[0].name
```

Comment:
can we maybe type the emit better? Or is that only possible with script setup?

Reply:
As discussed, seems only be possible with script setup.